ADMVEDU95/ADMVEDU95/Presentation/Detail/View/DetailViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  DetailVC.swift
+//  FirstWorkProject
+//
+//  Created by Satsishur on 12.04.2021.
+//
+
+import UIKit
+import AVFoundation
+import AVKit
+
+class DetailViewController: UIViewController {
+    //MARK: Views
+    lazy var songView: AudioDetailView = {
+        let view = AudioDetailView(frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height))
+        return view
+    }()
+    lazy var movieView: VideoDetailView = {
```

Comment:
songView and movieView should not be simultaneously in VC. As we want to reuse VC we should get a required view from presenter